swerve/swerveModule.py

```diff
@@ -1,4 +1,3 @@
-
 import math
```

Comment:
Was this file cleaned up on purpose? I see no functional changes?

Reply:
was not intentional, syncing code with main and other branches may have caused that to happen.